src/app/services/profile/transaction-information/transaction-information.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { LocalStorageService } from "src/app/services/local-storage.service";
+import { TestBed } from "@angular/core/testing";
+import {
+  HttpTestingController,
+  HttpClientTestingModule
+} from "@angular/common/http/testing";
+import { TransactionInformationService } from './transaction-information.service';
```

Comment:
hey @Jaman-dedy,  Let's use absolute imports here

Reply:
I worked on that @Tevinthuku